### DISCUSSION TITLE
Hi, my account has been flagged

### DISCUSSION THREAD

**Discussion by pgupta500** [community/community#101743] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey! As the title suggests, i think that my account has been flagged. Can you please unflag it and please let me know what can be done by me in order to prevent this in the future?

**Comment by samus-aran** [comment]
Hi @Coder321P 

This message can be an indication of many things that only the Support team can answer and resolve. So please do contact [Account or billing issues]([ https://support.github.com/request ](https://support.github.com/request)) Support and they will explain the details of this message.

**Comment by HalilBugday** [comment]
If you contact the github support team, they will help you. Enjoy your work. https://support.github.com

**Reply by xyz899** [reply]
so why Do I see everywhere that they never answer for it and some have to make new accounts again because of that
